curriculum/3-core/5-css-layout.md

```diff
@@ -74,6 +74,8 @@ Resources:
 
 - [Stacking context](https://developer.mozilla.org/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)
 
+<scrim-inline url="https://v2.scrimba.com/the-frontend-developer-career-path-c0j/~0d5" scrimtitle="Relative and Absolute Position"></scrim-inline>
```

Comment:
On reflection, I think this one feels a bit out of place (I appreciate you've put it where I asked you to!)

Instead, can you put it below the "Setting the positioning context of a positioned element by positioning an ancestor element." line?